docs/ref/django-admin.txt

```diff
@@ -1255,7 +1255,9 @@ files is:
 
 - Any option passed to the ``startapp`` command (among the command's supported
   options)
-- ``app_name`` -- the app name as passed to the command
+- ``app_python_path`` -- full python path to the application as passed to the command
```

Comment:
This should contain more details, e.g.
```
``app_python_path`` -- python path of the newly created app if it's below the current working directory, or an ``app_name`` otherwise.
```